src/management-system-v2/app/(dashboard)/iam/roles/role-page.tsx

```diff
@@ -47,10 +42,16 @@ const RolesPage: FC = () => {
     (role) => !ability.can('delete', toCaslResource('Role', role)),
   );
 
-  async function deleteRoles(userIds: string[]) {
-    setSelectedRowKeys([]);
-    setSelectedRows([]);
-    await Promise.allSettled(userIds.map((id) => deleteRole({ params: { path: { id } } })));
+  async function deleteRoles(roleIds: string[]) {
+    try {
+      await serverDeleteRoles(roleIds);
+
+      setSelectedRowKeys([]);
+      setSelectedRows([]);
+      router.refresh();
+    } catch (e) {
+      messageApi.error((e as Error).message);
```

Comment:
See other comment. This will always be a generic message in production.

Reply:
Fixed in one of my last commits